modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase

```diff
@@ -107,4 +107,40 @@ definition {
 		}
 	}
 
+	@description = "LPS-155308. Assert one filter can be removed."
+	@priority = "5"
+	test OneFilterCanBeRemoved {
+		task ("When there is a filter already established") {
+			AssertElementPresent(
+				key_filter = "Status: Approved, Draft",
+				locator1 = "FrontendDataSet#FILTER_SUMMARY_LABEL");
+		}
+
+		task ("Then it can be removed one by one") {
+			ClickNoError(
+				key_filter = "Status: Approved, Draft",
+				locator1 = "FrontendDataSet#FILTER_SUMMARY_LABEL");
+
+			FDSFilters.disableStatusFilters(key_status = "Approved");
+
+			Click(locator1 = "FrontendDataSet#EDIT_FILTER_BUTTON");
```

Comment:
@Esther-OC For the requirement "Then it can be removed one by one", do you mean 
- edit the filter to remove one by one (status filter contains both approved and draft > remove approved then remove draft) OR
- click on the close button to remove the filter summary label (removes both approved and draft at once)

Reply:
I have the same question as you. So when we specify the step **remove one by one**. I will fix the test and re-pull it to you.

Reply:
Hi!
For the "remove the filters one by one" I mean to click on the "x" icon included in the filter button to close/remove it.    So, in the current case, we will remove "approved and draft" at once as Susan said, and, if we have more filters enabled, we can remove filter1, filter 2, and leave filter 3 for instance.

At least this is what I understood, the filter is "status", other filter can be "date range" which will add other filter button with its "x" icon to remove it.  

Does it make sense to you?  ;-)

Reply:
Yep makes much more sense. Thanks! 

@WYZ-Issac can you update the test with this new behavior? I think for `When there are several filters already established` we will want to add a date filter as well. But it will probably throw an error because of [LPS-156836](https://issues.liferay.com/browse/LPS-156836). But it will still be added to the summary.

Reply:
OK, I will update the test.